examples/other/spark/Dockerfile

```diff
@@ -11,29 +11,23 @@ RUN touch /usr/bin/ch-ssh
 
 # Download and install Spark.
 #
```

Comment:
Explain three things in this comment block:

1. Why we aren't using `SPARK_NO_DAEMONIZE`.
2. Why we are using `/opt/spark`.
3. That we disapprove of the master/slave terminology, but that's what the scripts are called, so we have to use it. (This is true, right? Are there better aliases?)